.github/workflows/charts.yml

```diff
@@ -50,8 +51,6 @@ jobs:
       - name: Build and push images
         uses: docker/build-push-action@v4
         with:
```

Comment:
```suggestion
        with:
          cache-from: type=gha
          cache-to: type=gha,mode=max
          context: .
```

Reply:
Hey @isavov thanks for this, I made the change, let's hope it works.

Reply:
Seems it does. Thanks @isavov